From 50624e06e88aabd7ae123ad6a4495630f81cd706 Mon Sep 17 00:00:00 2001
From: Allen Winter <winter@kde.org>
Date: Sat, 16 Apr 2016 13:25:46 -0400
Subject: [PATCH 10/12] polkitqtlistener.cpp - pedantic

---
 agent/polkitqtlistener.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/agent/polkitqtlistener.cpp b/agent/polkitqtlistener.cpp
index dc5fdda..59f8d5d 100644
--- a/agent/polkitqtlistener.cpp
+++ b/agent/polkitqtlistener.cpp
@@ -59,7 +59,7 @@ static gboolean polkit_qt_listener_initiate_authentication_finish(PolkitAgentLis
         GAsyncResult         *res,
         GError              **error);
 
-G_DEFINE_TYPE(PolkitQtListener, polkit_qt_listener, POLKIT_AGENT_TYPE_LISTENER);
+G_DEFINE_TYPE(PolkitQtListener, polkit_qt_listener, POLKIT_AGENT_TYPE_LISTENER)
 
 static void polkit_qt_listener_init(PolkitQtListener *listener)
 {
-- 
2.17.0

